framerail/src/routes/[slug]/[...extra]/page.svelte

```diff
@@ -789,6 +1006,428 @@
   </div>
 {/if}
 
+{#if showFiles}
+  <div class="file-panel">
+    <div class="action-row file-action">
+      <button
+        class="action-button upload-file clickable"
+        type="button"
+        on:click|stopPropagation={() => {
+          showFileUploadAction = true
+        }}
+      >
+        {$page.data.internationalization?.upload}
+      </button>
+      <button
+        class="action-button deleted-file clickable"
+        type="button"
+        on:click={() => {
+          showFiles = false
+          getFileList(true).then(() => {
+            showFiles = true
+          })
+        }}
+      >
+        {$page.data.internationalization?.restore}
+      </button>
+    </div>
+
+    {#if fileMap.size > 0}
+      <div class="file-list">
+        <div class="file-list-header">
+          <div class="file-attribute name">
+            {$page.data.internationalization?.["wiki-page-file.name"]}
+          </div>
+          <div class="file-attribute created-at">
+            {$page.data.internationalization?.["wiki-page-file.created-at"]}
+          </div>
+          <div class="file-attribute updated-at">
+            {$page.data.internationalization?.["wiki-page-file.updated-at"]}
+          </div>
+          <div class="file-attribute licensing">
+            {$page.data.internationalization?.["wiki-page-file.license"]}
+          </div>
+          <div class="file-attribute mime">
+            {$page.data.internationalization?.["wiki-page-file.mime"]}
+          </div>
+          <div class="file-attribute size">
+            {$page.data.internationalization?.["wiki-page-file.size"]}
+          </div>
+          <div class="file-attribute action" />
+        </div>
+        {#each [...fileMap].sort((a, b) => b[0] - a[0]) as [_, file] (file.file_id)}
```

Comment:
I assume the 0 index here is the filename or something?

Reply:
Using the spread operator on a map gives an iterator of `[key, value]`, so in this case `[0]` refers to the key, which is the file id, so this each block sorts by file id in descending order.